### PR TITLE
Match wildcard service

### PR DESCRIPTION
**Pull request by jsravn**
So we can match kernel logs, etc.

### CODE REVIEW

File: alerter.go [hunk 1]
```diff
@@ -161,19 +161,29 @@ type JournalCtrl struct {
 func (this JournalCtrl) Logs(unit string) (string, error) {
 	timeout := time.Duration(1) * time.Second
 	searchPeriod := time.Duration(-this.duration) * time.Second
-	r, err := sdjournal.NewJournalReader(sdjournal.JournalReaderConfig{
-		Since: searchPeriod,
-		Matches: []sdjournal.Match{
+
+	var matches []sdjournal.Match
+	if unit == "*" {
```
Comment (sebbonnet) [inline alerter.go]: Could we not simply initialise the array when unit != "*",?

**Comment by jsravn** [comment]
@sebbonnet I did have a test, but had to remove it because it doesn't work on travis. The code I changed is stubbed out by the existing unit test so I can't create a unit test either.

File: alerter.go (hunk 1 above)
Comment (jsravn) [inline alerter.go]: Sure

**Comment by jsravn** [comment]
alerter_test.go is a bit of a mess. It should be stubbing out systemd at a much lower level than it does (it mocks the Logs method which has a lot of logic in it). I can have a look at improving it.

**Comment by jsravn** [comment]
@sebbonnet Okay, added a test based on our discussion.

**Pull request merged**